### DISCUSSION TITLE
Do you have plan to port Huada MCU，such as HC32F460

### DISCUSSION THREAD

**Discussion by wyl2008056** [hathach/tinyusb#1517] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Comment by hathach** [comment]
This is the first time I heard of this mcu. Short answer is "No" since I have no devkit. Though this may change if the mcu is popular and/or interesting enough.

**Reply by wyl2008056** [reply]
hi, hathach, thank you for your reply, HC32F460 mcu is popular  in chaina;  It is based on ARM Cortex-M4 32-bit RISC CPU;
i have the devkit , i can test the code on it.

**Reply by hathach** [reply]
Unfortunately I don’t have any devkit for HC mcu to begin with. If enough people find this interesting, I may try to order 1
